api/src/schemas.ts

```diff
@@ -326,6 +326,26 @@ export const schemas = {
       })
     }
   },
+  reportUser: {
+    body: Type.Object({
+      username: Type.String(),
+      reportDescription: Type.String()
```

Comment:
```suggestion
      username: Type.String({ minLength: 1 }),
      reportDescription: Type.String({ minLength: 1 })
```
So the `''` case is excluded by validation.